charts/folder-config-connector/templates/folder.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.name (not .Values.folders) }}
+---
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Folder
+metadata:
+  labels:
+  {{- include "folder-config-connector.labels" . | nindent 4 }}
+    {{- if .Values.labels }}
+    {{- toYaml .Values.labels | nindent 4 }}
+  {{- end  }}
+  {{- with .Values.annotations  }}
+  annotations:
+    {{- toYaml . | nindent 4 }}  
+  {{- end  }}
+  name: {{ .Values.name | quote }}
+spec:
+  {{- if .Values.displayName }}
+  displayName: {{ .Values.displayName | quote }}
+  {{- end }}
+  {{- if .Values.folderRef }}
+  folderRef:
+    {{- with .Values.folderRef }}
+    external: {{ .external | default nil | quote }}
```

Comment:
why do you sometimes use quote function? it is string value (from docs) so we shouldn't take care about the quotes. But if you want to protect against integers in string fields, please use quote function it everywhere.

Reply:
now usage of quote is consistent